src/common/options.cc

```diff
@@ -5414,8 +5414,17 @@ std::vector<Option> get_global_options() {
 
     Option("crimson_osd_scheduler_concurrency", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(0)
-    .set_description("The maximum number concurrent IO operations, 0 for unlimited")
+    .set_description("The maximum number concurrent IO operations, 0 for unlimited"),
 
+    // ----------------------------
+    // blk specific options
+    Option("block_device_type", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_description("Explicitly set the device type to select the driver if it's needed")
+    .set_long_description("The relationship between this configuration and driver is: "
+        "For device to use AIO driver dev_aio : KernelDevice, "
+        "For device to use SPDK driver dev_spdk : NVMEDevice, "
+        "For Optane device dev_pmem : PMEMDevice, "
+        "For Zoned device dev_hmsmr: HMSMRDevice")
```

Comment:
the class names are not user friendly, might want to rephrase them like "aio" for using libaio backend, which is the default one. spdk for NVMe devices driven by SPDK, pmem for PMEM devices driven by PMDK, hmsmr for SMR devices.

Reply:
might want to s/hmsmr/hm_smr/ or use zoned, as hm_smr is for short for Host Managed SMR.

Reply:
I'll use "hm_smr"

Reply:
Done